### ISSUE TITLE
Specify Custom SMTP Port

### ISSUE THREAD

**Issue by s6admin** [warrenbuckley/CWS-Start#26]
Any way to easily modify the port number being used with an SMTP server? For example, using smtp.gmail.com typically requires a port of 587.

Thank you!

**Comment by s6admin** [comment]
We've implemented this (along with the option to enable SSL) in our demo project and modified the Home DocType to allow Umbraco Users to customize these values. We'd be happy to contribute these to the project if you'd like.

**Comment by warrenbuckley** [comment]
Yes please a pull request for this changes sounds lovely please :)
